tests/lib/e2e/buckets/files.e2e-spec.ts

```diff
@@ -0,0 +1,347 @@
+import { ObjectId } from 'mongodb'
+import crypto from 'crypto'
+import axios from 'axios';
+import { engine, testServer } from '../setup';
+import { type User } from '../users.fixtures';
+import { createTestUser, getAuth, shutdownEngine } from '../utils';
+import sinon from 'sinon';
+
+
+describe('Bridge E2E Tests', () => {
+
+  let testUser: User
+  let axiosGetStub: sinon.SinonStub
+  beforeAll(async () => {
+    testUser = await createTestUser()
+
+    // Create fake contact per each node
+    const nodeIDs = Object.values(engine._config.application.CLUSTER)
+    await Promise.all(
+      nodeIDs.map((nodeID, index) => {
+        const payload = { nodeID, protocol: "1.2.0-INXT", address: `72.132.43.${index}`, port: 43758 + index, lastSeen: new Date(), }
+        return new Promise(resolve => engine.storage.models.Contact.record(payload, resolve))
+      })
+    )
+
+  })
+
+  afterAll(async () => {
+    await shutdownEngine(engine)
+  })
+
+
+
+  beforeEach(() => {
+
+    axiosGetStub = sinon.stub(axios, 'get')
+    jest.clearAllMocks()
+    axiosGetStub.callsFake(async (url: string) => {
+      if (url.includes('/v2/upload/link')) return { data: { result: 'http://fake-url' } }
+      if (url.includes('/v2/download/link')) return { data: { result: 'http://fake-url' } }
+      if (url.includes('exists')) return { status: 200 }
+    })
+  })
+
+  afterEach(() => {
+    axiosGetStub.restore()
+  })
+
+  describe('File Management v2', () => {
+
+    describe('Uploading a file', () => {
+
+      it('When a user wants to upload a file, it should work for owned buckets and get a list of upload links one per each file part', async () => {
+        // Arrange: Create a bucket
+        const { body: { id: bucketId } } = await testServer
+          .post('/buckets')
+          .set('Authorization', getAuth(testUser))
+          .expect(201)
+
+        // Act: start the upload
+        const response = await testServer.post(`/v2/buckets/${bucketId}/files/start`)
+          .set('Authorization', getAuth(testUser))
+          .send({ uploads: [{ index: 0, size: 1000, }, { index: 1, size: 10000, },], })
+
+        // Assert
+        expect(response.status).toBe(200);
+        const { uploads } = response.body;
+
+        expect(uploads).toHaveLength(2);
+
+        const upload1 = uploads[0];
+        expect(upload1.url).toBe('http://fake-url')
```

Comment:
Use a constant for storing the URL, as you have it duplicated around the code. If you want to change it, you should need to change it a lot of times, use a variable